agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/AiComponentInstaller.java

```diff
@@ -298,6 +310,18 @@ private static ApplicationInsightsXmlConfiguration buildXmlConfiguration(Configu
         return xmlConfiguration;
     }
 
+    //assuming that every other element is not needed - such as "type", "worker", "name"
+    //assuming same set of delimiters in every object name
+    static String getJmxDisplayName(String attribute) {
+        String[] nameParts = attribute.split("=|:|,");
```

Comment:
can you add a test with one of these separators inside of double quotes? I think you may need more complex algorithm for this, see https://github.com/glowroot/glowroot/blob/master/common2/src/main/java/org/glowroot/common2/repo/util/Gauges.java#L124-L189

Reply:
@trask Decided to try another approach... See if you approve. Basically, I was thinking that too if we include the domain, type and name, followed by attribute it will be more readable. Using ObjectName for parsing. 
Questions:
1. Are you ok with using ObjectName class? In case the objectname format may change in the future, this is probably a safe option to go with in the longer run?
2. Is the format I described above adequate? 
3. If point 1. is ok, but point 2. is not, I can keep using ObjectName but modify how I extract the properties.